amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-xml-maven.adoc

```diff
@@ -1,18 +1,15 @@
 = Amazon S3 Connector 5.7 XML and Maven Support
 :page-aliases: connectors::amazon/amazon-s3-connector-xml-maven.adoc
 
-If you manually code a Mule app in XML either from the
-Studio XML editor or a text editor, perform the
-following actions to get access to the connector:
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio:
 
-. Add the namespace for the connector to your XML flow.
-. Add dependency information to your Maven `pom.xml` file.
+If you manually code a Mule runtime engine (Mule) app in XML, either from the Anypoint Studio XML editor or from a text editor, you can access the connector from your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
 
-In the Studio visual editor, when you drag a connector from the palette to the canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+If you add the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location and it also adds a dependency to the `pom.xml` file.
 
 == Add a Namespace for the Connector
 
-Paste the following code inside the <mule> tag of the header of your configuration XML:
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
```

Comment:
In the piece about "Add a POM File Dependency," please use the text from the template, including using <version>x.x.x</version> instead of RELEASE. Apparently, using RELEASE caused a security vulnerability.